core/src/main/scala/org/apache/spark/deploy/ExternalShuffleService.scala

```diff
@@ -169,6 +169,7 @@ object ExternalShuffleService extends Logging {
     Utils.initDaemon(log)
     val sparkConf = new SparkConf
     Utils.loadDefaultSparkProperties(sparkConf)
+    Logging.uninitialize()
```

Comment:
shall we add comments to explain why uninitialize?

Reply:
thanks for your advice, updated.